tests/model_hub_tests/pytorch/test_sampling.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018-2024 Intel Corporation
```

Comment:
don't think `test_sampling` is lucky name. Any other options better reflects idea of the test?
`test_apply_pat`?

Reply:
test_pa_transformation

Reply:
We'll go with _test_pa_transformation_